framework/include/task_manager/task_manager.h

```diff
@@ -232,18 +245,21 @@ int task_manager_resume(int handle, int timeout);
 int task_manager_restart(int handle, int timeout);
 /**
  * @brief Request to send messages to the task
- * @details @b #include <task_manager/task_manager.h>
+ * @details @b #include <task_manager/task_manager.h>\n
+ * task_manager_unicast can be used with SYNC and ASYNC mode.\n
+ * With SYNC mode, reply msg can be sent through task_manager_reply_unicast API.
  * @param[in] handle the handle id of task to be sent
- * @param[in] msg message to be unicasted
- * @param[in] msg_size the size of message
+ * @param[in] send_msg message structure to be unicasted
+ * @param[out] reply_msg the reply msg structure for sync. If this is NULL, task_manager_unicast works asynchronously.
```

Comment:
I think providing information that reply_msg is needed to be alloc before using this API is better.

Reply:
alloc is not needed. it is okay.